lib/pbench/test/functional/server/test_put.py

```diff
@@ -45,7 +45,7 @@ def test_index_all(self, server_client: PbenchServerClient, login_user):
         # For each dataset we find, poll the state until it reaches Indexed
         # state, or until we time out.
         start = time.time()
-        timeout = start + (60.0 * 5.0)
+        timeout = start + (60.0 * 10.0)
```

Comment:
Not that I object... but I'm curious. I've never seen a timeout except where something's broken that will never resolve. Have you? If so, what happened?

Reply:
This I made as a separate commit when I noticed the functional tests failures, but could not find anything in the logs to indicate why.  I figured a real hang would still fail with the extension, and a flaky test would just pass.

I'm happy to revert this change.